src/main/java/com/targomo/client/api/TravelOptions.java

```diff
@@ -124,24 +124,28 @@
     @Transient private Integer multiGraphSerializationMaxGeometryCount                       = null;
     @Transient private MultiGraphAggregationType multiGraphAggregationType                   = null;
     @Transient private Boolean multiGraphAggregationIgnoreOutlier                            = null;
-    @Transient private Integer multiGraphAggregationOutlierPenalty                           = null;
+    @Transient private Float multiGraphAggregationOutlierPenalty                             = null;
     @Transient private Double multiGraphAggregationMinSourcesRatio                           = null;
     @Transient private Integer multiGraphAggregationMinSourcesCount                          = null;
     @Transient private Double multiGraphAggregationMaxResultValueRatio                       = null;
-    @Transient private Integer multiGraphAggregationMaxResultValue                           = null;
+    @Transient private Float multiGraphAggregationMaxResultValue                             = null;
     @Transient private Set<String> multiGraphAggregationFilterValuesForSourceOrigins         = null;
-    @Transient private Map<String, SourceParameter> multiGraphSourceParameters               = null;
-    @Transient private Map<String, AggregationConfig> multiGraphPreAggregationPipeline       = null;
+    @Transient private Double multiGraphAggregationGravitationExponent                       = null;
+    @Transient private LinkedHashMap<String, AggregationInputParameters> multiGraphAggregationInputParameters = null;
+    @Transient private LinkedHashMap<String, AggregationConfiguration> multiGraphPreAggregationPipeline = null;
     @Transient private MultiGraphLayerType multiGraphLayerType                               = null;
     @Transient private MultiGraphLayerEdgeAggregationType multiGraphLayerEdgeAggregationType = null;
     @Transient private Integer multiGraphLayerGeometryDetailPerTile                          = null;
     @Transient private Integer multiGraphLayerMinGeometryDetailLevel                         = null;
     @Transient private Integer multiGraphLayerMaxGeometryDetailLevel                         = null;
     @Transient private Integer multiGraphLayerGeometryDetailLevel                            = null;
+    @Transient private Integer multiGraphLayerStatisticGroupId                               = null;
+    @Transient private List<Short> multiGraphLayerStatisticsIds                              = null;
     @Transient private Integer multiGraphTileZoom                                            = null;
     @Transient private Integer multiGraphTileX                                               = null;
     @Transient private Integer multiGraphTileY                                               = null;
-
+    @Transient private Float multiGraphPostAggregationFactor                                   = null;
```

Comment:
rename: `multiGraphAggregationPostAggregationFactor` (also all the methods, toString and method parameters)